content/en/docs/concepts/workloads/controllers/job.md

```diff
@@ -14,16 +14,16 @@ weight: 50
 <!-- overview -->
 
 A Job creates one or more Pods and will continue to retry execution of the Pods until a specified number of them successfully terminate.
-As pods successfully complete, the Job tracks the successful completions.  When a specified number
-of successful completions is reached, the task (ie, Job) is complete.  Deleting a Job will clean up
+As pods are successfully completed, the Job tracks the successful completions.  When a specified number
+of successful completions is reached, the task (i.e. Job) is complete.  Deleting a Job will clean up
 the Pods it created. Suspending a Job will delete its active Pods until the Job
 is resumed again.
 
 A simple case is to create one Job object in order to reliably run one Pod to completion.
 The Job object will start a new Pod if the first Pod fails or is deleted (for example
 due to a node hardware failure or a node reboot).
 
-You can also use a Job to run multiple Pods in parallel.
+You can also use a Job to run multiple Pods in parallel. See [this](https://kubernetes.io/docs/tasks/job/coarse-parallel-processing-work-queue/#defining-a-job).
```

Comment:
```suggestion
You can also use a Job to run multiple Pods in [parallel](/docs/concepts/workloads/controllers/job/#parallel-jobs). 
```
There are multiple types of parallel jobs, the one suggested is for a work queue.
I suggest to add the links to the examples in the section that describes parallels jobs: https://kubernetes.io/docs/concepts/workloads/controllers/job/#parallel-jobs
You can add a line with the example for the work queue parallel job